app/helpers/cms_helper.rb

```diff
@@ -150,4 +150,40 @@ def cta_api
   def cta_live_report
     @cta_live_report ||= CallToAction.find_by_css_class('live-report')
   end
+
+  def get_resource_links 
+    links = []
+
+    if get_resource_link then links.push(get_resource_link) end
+    if get_resource_file then links.push(get_resource_file) end
```

Comment:
I reckon this could be potentially rewritten to combine the `get_resource_links`, `get_resource_link` and `get_resource_file` into one method as I see that there's repeated code.